packages/database/src/migrations/20210622234149-LESMISStudentCountToStackedBar-modifies-data.js

```diff
@@ -0,0 +1,200 @@
+'use strict';
+
+import { insertObject, generateId, findSingleRecord } from '../utilities';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+const CODE = 'LESMIS_student_gender_stacked_gpi';
+
+const REPORT_CONFIG = {
+  fetch: {
+    dataElements: [
+      'nostu_p1_f',
+      'nostu_p1_m',
+      'nostu_p2_f',
+      'nostu_p2_m',
+      'nostu_p3_f',
+      'nostu_p3_m',
+      'nostu_p4_f',
+      'nostu_p4_m',
+      'nostu_p5_f',
+      'nostu_p5_m',
+      'nostu_s1_f',
+      'nostu_s1_m',
+      'nostu_s2_f',
+      'nostu_s2_m',
+      'nostu_s3_f',
+      'nostu_s3_m',
+      'nostu_s4_f',
+      'nostu_s4_m',
+      'nostu_s5_f',
+      'nostu_s5_m',
+      'nostu_s6_f',
+      'nostu_s6_m',
+      'nostu_s7_f',
+      'nostu_s7_m',
+    ],
+    aggregations: [
+      {
+        type: 'SUM_PER_ORG_GROUP',
```

Comment:
We need a `MOST_RECENT` first, so that we get the most recent submission per school rather than summing across multiple data points